### PR TITLE
[ Docs] translated peft (ES) and toctree.yml add

### PR DESCRIPTION
**Pull request by njackman-2344**
Add the Spanish version of `peft.md` to `transformers/docs/source/es` 

#28936

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu @aaronjimv  If you don't mind reviewing, thanks!! :)

### CODE REVIEW

File: docs/source/es/_toctree.yml [hunk 1]
```diff
@@ -96,6 +96,8 @@
     title: Mecanismos de atención
   - local: pad_truncation
     title: Relleno y truncamiento
+    local: peft
```
Comment (stevhliu) [inline docs/source/es/_toctree.yml:99]: This should be in the "Tutorial" section.

```suggestion
   - local: peft
```

File: docs/source/es/peft.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+[[open-in-colab]]
+
+[Parameter-Efficient Fine Tuning (PEFT)](https://huggingface.co/blog/peft) métodos congelan los parámetros del modelo preentrenado durante el afinamiento y agregan un pequeño número de parámetros entrenables (los adaptadores) encima de eso.  Varios parámetros entrenables (los adaptadores) están entrenados para aprender la tarea específica.  Está manera a sido comprobada en ser eficiente con la memoria y con menos uso de computar mientras produciendo resultados comparable a un modelo completamente afinado. 
+
+ Adaptadores entrenados con PEFT también son usualmente un orden de magnitud más pequeños que los modelos enteros, haciéndolos más convenientes para compartir, archivar, y cargar. 
+
+<div class="flex flex-col justify-center">
+  <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/peft/PEFT-hub-screenshot.png"/>
+  <figcaption class="text-center">The adapter weights for a OPTForCausalLM model stored on the Hub are only ~6MB compared to the full size of the model weights, which can be ~700MB.</figcaption>
```
Comment (stevhliu) [inline docs/source/es/peft.md:22]: Would be nice to also translate the caption 🙂

File: docs/source/es/peft.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+peft_model_id = "ybelkada/opt-350m-lora"
+
+model = AutoModelForCausalLM.from_pretrained(model_id)
+model.load_adapter(peft_model_id)
+```
+
+## Cargar en 8bit o 4bit
+
+La integración de `bitsandbytes` apoya los tipos de datos precisos que son utilizados para cargar modelos grandes porque
+ guarda memoria (mira la [guia](./quantization#bitsandbytes-integration) de `bitsandbytes` para aprender mas). Agrega el parametro `load_in_8bit` o el parametro `load_in_4bit` al [`~PreTrainedModel.from_pretrained`] y coloca `device_map="auto"` para effectivamente distribuir el modelo tu hardware:
```
Comment (stevhliu) [inline docs/source/es/peft.md:87]: The "Quantization" doc doesn't exist in Spanish yet so let's link to the English one for now

```suggestion
 guarda memoria (mira la [guia](https://huggingface.co/docs/transformers/quantization#bitsandbyes) de `bitsandbytes` para aprender mas). Agrega el parametro `load_in_8bit` o el parametro `load_in_4bit` al [`~PreTrainedModel.from_pretrained`] y coloca `device_map="auto"` para effectivamente distribuir el modelo tu hardware:
```

**Review by stevhliu — COMMENTED** [review]
Thanks for adding! 🤗

**Review by aaronjimv — COMMENTED** [review]
Hi, thanks for adding! The translation in general is good, but I have to point out that it presents several issues with fluency (making it sound natural) and coherence (making it reflect the original content).

**Review by aaronjimv — CHANGES_REQUESTED** [review]
Hi @njackman-2344, I appreciate you asking me to review again. However, I still have some concerns about the fluency and coherence of the translation. Would you mind taking another look at all the comments made [here](https://github.com/huggingface/transformers/pull/30554/commits/03a1a7d2c0b00ecfde6f4b9c3c19b68ae341b5d7) and addressing those issues? I think it would really make a big difference. Additionally, I wanted to suggest you to use tools like [HuggingChat](https://huggingface.co/chat/) to assist you in the translations and improving their fluency and coherence. Once you've made the necessary changes, I'll be happy to review again.

**Comment by njackman-2344** [comment]
> Hi @njackman-2344, I appreciate you asking me to review again. However, I still have some concerns about the fluency and coherence of the translation. Would you mind taking another look at all the comments made [here](https://github.com/huggingface/transformers/pull/30554/commits/03a1a7d2c0b00ecfde6f4b9c3c19b68ae341b5d7) and addressing those issues? I think it would really make a big difference. Additionally, I wanted to suggest you to use tools like [HuggingChat](https://huggingface.co/chat/) to assist you in the translations and improving their fluency and coherence. Once you've made the necessary changes, I'll be happy to review again.

Ok! Thank you! I didn't see the corrections. I will do that.

**Pull request closed without merging**